lib/stacks/api/appConfiguration.ts

```diff
@@ -54,6 +54,10 @@ export class AppConfigurationApiStack extends Stack {
                 description: 'Update the MLSpace application configuration',
                 path: 'app-config',
                 method: 'POST',
+                environment: {
+                    ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN: props.endpointConfigInstanceConstraintPolicy.managedPolicyArn,
```

Comment:
Why are we doing this here? Shouldn't this be populated in the env_variables from the `mlspace_config`?

Reply:
In `lib/utils/apiFunction` there's this block:

```
environment: {
            DATASETS_TABLE: mlspaceConfig.DATASETS_TABLE_NAME,
            PROJECTS_TABLE: mlspaceConfig.PROJECTS_TABLE_NAME,
            PROJECT_USERS_TABLE: mlspaceConfig.PROJECT_USERS_TABLE_NAME,
            USERS_TABLE: mlspaceConfig.USERS_TABLE_NAME,
            RESOURCE_SCHEDULE_TABLE: mlspaceConfig.RESOURCE_SCHEDULE_TABLE_NAME,
            RESOURCE_METADATA_TABLE: mlspaceConfig.RESOURCE_METADATA_TABLE_NAME,
            APP_CONFIGURATION_TABLE: mlspaceConfig.APP_CONFIGURATION_TABLE_NAME,
            SYSTEM_TAG: mlspaceConfig.SYSTEM_TAG,
            MANAGE_IAM_ROLES: mlspaceConfig.MANAGE_IAM_ROLES ? 'True' : '',
            NOTEBOOK_ROLE_NAME: notebookRoleName,
            PERMISSIONS_BOUNDARY_ARN: permissionsBoundaryArn || '',
            ...funcDef.environment,
            ...mlspaceConfig.ADDITIONAL_LAMBDA_ENVIRONMENT_VARS,
        },
```

Where we give most lambdas some baseline env vars... but since these only apply to this lambda, this seems like the perfect place for them to be defined.